first_task.scala

```diff
@@ -0,0 +1,260 @@
+// Databricks notebook source
+import java.sql.Timestamp
+import java.net.{URI, URISyntaxException}
+
+
+import org.apache.spark.sql.functions._
+import org.apache.spark.sql.{Dataset, Encoders}
+import org.apache.spark.sql.types.DateType
+import org.jsoup.Jsoup
+
+
+import spark.implicits._ //!!!
+
+
+// COMMAND ----------
+
+case class Question(id: Long,
+                    ownerUserId: Long,
+                    creationDate: Timestamp,
+                    score: Long,
+                    title: String,
+                    body: String)
+
+val questionsDs = spark.read
+      .option("multiLine", true)
+      .option("header", true)
+      .schema(Encoders.product[Question].schema)
+      .option("escape", "\"")
+      .csv("/FileStore/tables/Questions.csv")
+      .na.drop("all")
+      .as[Question]
+      
+questionsDs.cache()
+
+
+// COMMAND ----------
+
+case class Answer(id: Long,
+                  ownerUserId: Long,
+                  creationDate: Timestamp,
+                  partnerId: Long,
+                  score: Long,
+                  body: String)
+
+val answersDs = spark.read
+      .option("multiLine", true)
+      .option("header", true)
+      .schema(Encoders.product[Answer].schema)
+      .option("escape", "\"")
+      .csv("/FileStore/tables/Answers.csv")
+      .na.drop("all")
+      .as[Answer]
+
+answersDs.cache()
```

Comment:
I think this can be chained, i.e.
```
val aDF = spark 
  .read                       
  .parquet(parquetFile)       
  .cache                     
```